src/display3d.cpp

```diff
@@ -3366,6 +3366,7 @@ static void renderSurroundings(const glm::mat4 &viewMatrix)
 /// Smoothly adjust player height to match the desired height
 static void trackHeight(float desiredHeight)
 {
+	desiredHeight = (int)(desiredHeight / HEIGHT_TRACK_INCREMENTS) * HEIGHT_TRACK_INCREMENTS;
```

Comment:
Instead of integer truncation, might be worth doing something like:
```suggestion
	desiredHeight = std::ceil(desiredHeight / HEIGHT_TRACK_INCREMENTS) * HEIGHT_TRACK_INCREMENTS;
```

Reply:
Thanks! I was thinking about something in the lines of this, but didn't know how to do it!